perf/istio/sds-certmanager/templates/fortio-client.yaml

```diff
@@ -0,0 +1,54 @@
+# Fortio load generator that simulates external traffic. Communicates with the
+# internal service via external LoadBalancer which IP address is supposed to be
+# passed via a config map.
+# Expects real DNS name and attempts to connect over https (allowing
+# non-trusted certificates so we can use "staging" version of "Let's Encrypt").
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: fortio-client-config
+data:
+  external_addr: istio-ingress-{{ .Values.namespace }}
+---
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: fortio-client
+spec:
+  template:
+    metadata:
+      labels:
+        app: fortio-cli-raw
+        version: v1
+      annotations:
+        sidecar.istio.io/inject: "false"
+    spec:
+      containers:
+      - name: fortio-client
+        image: {{ .Values.fortioImage }}
+        imagePullPolicy: Always
+        args:
+          - load
+          - -t
+          - "0"
+          - -abort-on
+          - "-1"
+          - -timeout
+          - "{{ .Values.clientConnTimeoutDuration }}"
+          - -c
+          - "{{ .Values.numOfClientConns }}"
+          - -qps
+          - "{{ .Values.numOfClientQps }}"
+          - -https-insecure
+          - "https://$(EXTERNAL_ADDR)/echo?size=1024"
+        env:
+          - name: EXTERNAL_ADDR
+            valueFrom:
```

Comment:
Why can't we just use `value: {{ .Values.ingressDomain }}` here?

Reply:
Yeah, you're right, this is copy-paste from "gateway-bouncer", we don't need all this ConfigMap manipulations in this scenario, as we have a real domain here. Pushed a fix.